packages/caqti-driver-pgx/caqti-driver-pgx.2.1.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Petter A. Urkedal <paurkedal@gmail.com>"
+authors: [
+  "Petter A. Urkedal <paurkedal@gmail.com>"
+]
+license: "LGPL-3 with OCaml linking exception"
```

Comment:
```suggestion
license: "LGPL-3 WITH OCaml linking exception"
```

Reply:
It's actually supposed to be "LGPL-3.0-or-later WITH LGPL-3.0-linking-exception".  Thanks for pointing it out.